lib/reverse_sentence.rb

```diff
@@ -1,6 +1,30 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) - the n is because the number of times the loop runs is dependant on the length of my_sentence.
+# Space complexity: O(n) - array created by split method. After reversing the words, join method used to make it into a single string again.
```

Comment:
The problem does ask you to do this "in-place" in other words with O(1) space complexity.  

The method also does not work.